app/components/BlogCard.tsx

```diff
@@ -0,0 +1,75 @@
+import { FunctionComponent } from "react";
+import dayjs from "dayjs";
+import cx from "classnames";
+import { PostOrPage } from "@tryghost/content-api";
+
+type Props = {
+  blogPost: PostOrPage;
+};
+
+const BlogCard: FunctionComponent<Props> = ({ blogPost }) => {
+  const tag = (blogPost.tags || [])[0];
+  const author = (blogPost.authors || [])[0];
+
+  return (
+    <div className="relative flex flex-col w-full" style={{ maxWidth: 400 }}>
+      <div className="flex-grow">
+        <a href={blogPost.url} target="_blank" className="mb-2">
+          <img
+            src={blogPost.og_image || ""}
+            alt="blog-card-img"
```

Comment:
**issue:** the alt text on this image isn't very helpful. Because this image is inside a link, it should probably say something like "Read this article on CodeSee Learn". [Here's a good primer on alt text for images](https://accessibility.huit.harvard.edu/describe-content-images).